app/components/settings/contact-info-section.js

```diff
@@ -0,0 +1,82 @@
+import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
+
+const { Component } = Ember;
+
+export default Component.extend(FormMixin, {
+
+  identification : '',
+  phone          : '',
+  isLoading      : false,
+
+  getValidationRules() {
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+      fields : {
+        identification: {
+          identifier : 'email',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter your email ID')
+            },
+            {
+              type   : 'email',
+              prompt : this.i18n.t('Please enter a valid email ID')
+            }
+          ]
+        },
+        phone: {
+          identifier : 'phone',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter your phone')
+            }
+          ]
+        }
+      }
+    };
+  },
+
+  actions: {
+    submit() {
+      this.onValid(() => {
+        let credentials = this.getProperties('identification', 'phone'),
+            authenticator = 'authenticator:jwt';
+
+        this.set('errorMessage', null);
+        this.set('isLoading', true);
+
+        this.get('session')
+          .authenticate(authenticator, credentials)
+          .then(() => {
+            this.get('loader').get('/users/me').then(data => {
+              this.get('session').set('data.currentUser', data);
+            });
+          })
+          .catch(reason => {
+            if (reason.hasOwnProperty('code') && reason.code === 401) {
+              this.set('errorMessage', this.i18n.t('Your credentials were incorrect.'));
+            } else {
+              this.set('errorMessage', this.i18n.t('An unexpected error occurred.'));
+            }
+            this.set('isLoading', false);
+          })
+          .finally(() => {
+            this.set('phone', '');
+          });
+      });
+    }
+  },
+
+  didInsertElement() {
```

Comment:
This whole function is not needed here. Please do not just copy from a different page and add things without trying to understand their purpose.